types/d3-array/index.d.ts

```diff
@@ -357,6 +357,10 @@ export function bisectRight(array: ArrayLike<number>, x: number, lo?: number, hi
 export function bisectRight(array: ArrayLike<string>, x: string, lo?: number, hi?: number): number;
 export function bisectRight(array: ArrayLike<Date>, x: Date, lo?: number, hi?: number): number;
 
+export function bisectCenter(array: ArrayLike<number>, x: number, lo?: number, hi?: number): number;
```

Comment:
I'm not going to comment on whether it's more beautiful or not, but using optional parameters makes the definitions a lot less repetitive which is more important than aesthetics in my opinion since it makes it easier to maintain.

Reply:
Agree. Splitting up to multiple lines would be too verbose and semantically awkward imo.